tests/chart_tests/test_prometheus_config_configmap.py

```diff
@@ -164,3 +164,46 @@ def test_prometheus_config_configmap_scrape_interval(self, kube_version):
             if job["job_name"] == "postgresql-exporter":
                 assert job["scrape_interval"] == "foo"
                 assert job["scrape_timeout"] == "bar"
+
+    def test_prometheus_config_configmap_with_node_exporter(self, kube_version):
+        """Validate the prometheus config configmap has the node-exporter enabled with params."""
+        doc = render_chart(
+            name="foo-name",
+            namespace="bar-ns",
+            kube_version=kube_version,
+            show_only=self.show_only,
+            values={
+                "global": {
+                    "nodeExporterEnabled": True,
+                },
+                "prometheus": {
+                    "nodeExporter": {"scrape_interval": "333s"},
+                },
+            },
+        )[0]
+
+        config_yaml = yaml.safe_load(doc["data"]["config"])
+        nodeExporterConfigs = [
+            x for x in config_yaml["scrape_configs"] if x["job_name"] == "node-exporter"
+        ]
+
+        if nodeExporterConfigs[0]["scrape_interval"]:
+            assert nodeExporterConfigs[0]["scrape_interval"] == "333s"
```

Comment:
We want this to always be set. The `if` here will cause a false positive test result if there is no `scrape_interval`.
```suggestion
        assert nodeExporterConfigs[0]["scrape_interval"] == "333s"
```